calculator.js

```diff
@@ -5,24 +5,70 @@ const exampleAdditionInput = {
   operation: 'add',
 }
 
+const checkNum = function (num) {
+  if (isNaN(num)) {
+    console.log(`${num} is not a number.  Try again.`)
+    return false
+  } else {
+    return true
+  };
+}
```

Comment:
Don't forget your semi-colons for javascript best practices :)

```suggestion
const checkNum = function (num) {
  if (isNaN(num)) {
    console.log(`${num} is not a number.  Try again.`);
    return false;
  } else {
    return true;
  };
}
```